lambda-notes/src/index.js

```diff
@@ -3,6 +3,17 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import firebase from 'firebase'
```

Comment:
Awesome job going for firebase.  One thing, you never want to make API keys public.  Its likely that google will notify you as well, but they should always be left out of your git repo.  Its generally best practices to set up a folder where you can set up a variable for the api key, and then import it into the file you need.  the folder that holds the keys should be ignored by adding it to you .gitignore file.